apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Labels.cs

```diff
@@ -27,6 +28,16 @@ namespace Google.Cloud.Diagnostics.AspNet
     /// </summary>
     internal static class Labels
     {
+        /// <summary>The version of the trace agent.</summary>
+        internal static readonly string AgentVersion = typeof(Labels)
+            .GetTypeInfo()
+            .Assembly
+            .GetCustomAttribute<AssemblyInformationalVersionAttribute>()
+            .InformationalVersion;
+
+        /// <summary>The name of the trace agent.</summary>
+        internal const string AgentName = "google-cloud-c-sharp-trace";
```

Comment:
Any reason for `c-sharp` rather than `csharp` here? I think that would be more generally expected.